.github/workflows/psql-release-notifier.yaml

```diff
@@ -0,0 +1,57 @@
+# #############################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+name: PostgreSQL new release notifier
+
+on:
+    workflow_dispatch:
+    schedule:
+        # Run every day at 01:00 at night
+        - cron: "0 1 * * *" # UTC time
+env:
+    TRACTUSX_MAILINGLIST: _CHANGE_ME_@MAIL.COM 
```

Comment:
Can't we add the actual mailing list address here? I don't think this is a secret

Reply:
Certainly not a secret but prevention from sending out emails after merge. Would like to get first official GO that we start to send that communication.